Framework/DataHandling/src/LoadBBY.cpp

```diff
@@ -23,9 +23,18 @@
 #include "MantidKernel/UnitFactory.h"
 #include "MantidNexus/NexusClasses.h"
 
+#include "boost/algorithm/string/trim.hpp"
```

Comment:
Do you mind changing the quote marks to `<>`?

Reply:
No problem, my oversight.

Reply:
@gvardany It appears there was a failure with the test system, the log for the OSX tests show that it all passed but there was a Jenkins notification failure that prevents it recognizing that the test is complete.